### DISCUSSION TITLE
테이블 테두리 및 배경색 설정

### DISCUSSION THREAD

**Discussion by damon-911** [pronist/hELLO#199] [Q&A] [answered]
테이블을 만들었는데 테두리와 배경색이 적용되지 않아서 혹시 해결법을 아시는지 여쭤봅니다.

<글>
![image](https://user-images.githubusercontent.com/79182122/220311383-a8d4c2e3-f544-4655-9313-4b85370515cf.png)

<실제>
![image](https://user-images.githubusercontent.com/79182122/220311721-f8e21ba3-db70-4664-b1b3-3cefbbf55c84.png)

**Comment by pronist** [comment, marked as answer]
@damon-911 테이블 테두리는 모두 하나로 통일되었습니다. 다크모드에서 보는 경우 독자가 읽기 힘든 경우가 있기 때문입니다.

**Reply by damon-911** [reply]
테두리가 아예 없게 나오는데 원래 저렇게 설정해 놓으신 건가요?

**Reply by pronist** [reply]
네, 그렇습니다. 셀 간에는 테두리 설정을 하지 않았습니다.

![1](https://user-images.githubusercontent.com/25624965/220631261-83afface-cb62-4dee-95ae-2315de63a2ec.jpg)

테두리 설정은 원본 소스에서 다음과 같이 정의되어 있습니다.

https://github.com/tistory-projects/tistory-skin-hELLO/blob/17f2d7d5e0e126d532087e36041585429f12272f/views/Main/Post/Permalink/Post/Content.pug#L371-L376

**Reply by damon-911** [reply]
넵 이해했습니다!
